src/SupportServiceProvider.php

```diff
@@ -17,7 +17,7 @@ public function configurePackage(Package $package): void
          */
         $package
             ->name('support')
-            ->hasConfigFile()
```

Comment:
I'm not sure if we can pass the name to this or if we need to handle it differently.

Reply:
It looks like it accepts the name of the config file. The tests are passing! ✅